packages/destination-actions/src/destinations/stackadapt/__tests__/__snapshots__/snapshot.test.ts.snap

```diff
@@ -0,0 +1,21 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Testing snapshot for actions-stackadapt destination: forwardEvent action - all fields 1`] = `""`;
+
+exports[`Testing snapshot for actions-stackadapt destination: forwardEvent action - required fields 1`] = `""`;
```

Comment:
There is one required field - `user_id` (which can be derived from either user ID or anonymous ID)

The snapshot files are all auto-generated by Segment. The data in the snapshot file is the expected request body and we don't have one because we make GET requests with all data passed in the query string.

Reply:
gotcha makes sense. would it be worthwhile snapshotting the expected get request with data in query string for some of the event types we explicitly support in our CDP ? I'm thinking it'd double as documentation for downstream teams that want to understand how we map segment events (i.e. the ecommerce events were supporting in Shopify v1 can be cross referenced with what's supported in segment) back into pixel service calls or alternatively we can maintain a separate guide/doc to explain the mapping strategy for different event types?

Reply:
No issue with putting the expected query strings for the `index.test.ts` file into a snapshot file. I don't think the tests or any other files in this repo should be used for documentation for our internal teams because it belongs to Segment and we will always need their permission to merge any changes upstream.

Reply:
On second thought I actually think it's safer to force developers to manually update the expected query params when they update the code rather than just regenerating a snapshot and hopefully reviewing it.